osquery/remote/enroll/plugins/tls_enroll.cpp

```diff
@@ -86,6 +86,8 @@ Status TLSEnrollPlugin::requestKey(const std::string& uri,
   boost::property_tree::ptree params;
   params.put<std::string>(FLAGS_tls_enroll_override, getEnrollSecret());
   params.put<std::string>("host_identifier", getHostIdentifier());
+  params.put<std::string>("platform",
```

Comment:
Technically this is `platform_type`, or `platform_mask`, do you want `kSDKPlatform` instead?